modules/ROOT/pages/event-tracking.adoc

```diff
@@ -186,29 +194,33 @@ services:
 ----
 
 
+=== Complex Tracking Example
+
+This example sets tracking levels and settings at all three levels: global, Mule app, and flow.
 
-=== Setting Complex Tracking Configurations
+Global:
```

Comment:
The narrative begins to look a bit like shorthand at this point. Will this make sense to our audience?